torch/_dynamo/output_graph.py

```diff
@@ -2229,6 +2199,296 @@ def maybe_lift_tracked_freevar_to_input(self, arg):
             return arg
         return self.lift_tracked_freevar_to_input(arg)
 
+    # See NOTE: [Auto lift basic free symbols when create_graph_input] for overall design
+    # This function should be used to track the unbacked symbols with its proxy created during
+    # dynamo tracing. It's invoked in wrap_fx_proxy for every proxied call for tensors and symbols.
+    # So that subgraph knows how to bind a symbol input with parent's proxy.
+    def lazy_bind_unbacked_symbols(self, example_value, e_proxy: torch.fx.Proxy):
```

Comment:
Can we just call this... "track_unbacked_symbols". "bind" and "lazy" are confusing by themselves, the full context is "we're going to track this unbacked symint and try hard not to create a proxy if it's not going to be used".

I want to call this "track" because this is the act of adding the symint to current_tracer.bound_symbols

Reply:
_lazy_bind_unbacked_symbols_in_tensors -> can this just be "track_unbacked_symbols" as well? track_unbacked_symbols can switch on the type of the input to figure out what it should do (torch.Tensor vs torch.SymInt).

Reply:
Also, you might want to put `lift_symbols_in_tensor` and `lift_symbols_in_symint` together as well, but up to you